docs/content/en/integrations/parsers/file/snyk_code.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Snyk"
```

Comment:
Hi @FelixHernandez 
title should then be "Snyk Code"

In general I personally like it more if we don't add more and more parsers for multiple functionalities of one vendor, but maybe rather one parser which covers all functionalities of a vendor. This is why I made PRs like these two:
https://github.com/DefectDojo/django-DefectDojo/pull/9522
https://github.com/DefectDojo/django-DefectDojo/pull/9355

Furthermore: Did you check out this PR:
https://github.com/DefectDojo/django-DefectDojo/pull/9289